README.md

```diff
@@ -33,11 +33,85 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
 - [List of hosted mainnets, testnets, and additional chains using BlockScout](https://docs.blockscout.com/for-projects/supported-projects)
 - [Hosted instance versions](https://docs.blockscout.com/about/use-cases/hosted-blockscout)
 
-
 ## Getting Started
 
+1. Install requirements
+
+    For a complete list of requirements, see the [blockscout docs](https://docs.blockscout.com/for-developers/information-and-settings/requirements).
```

Comment:
to get elixir command i had to modify my PATH in my zsh file 

i did `asdf which elixir` cut off the end and then copied that to zshrc like PATH=/$HOME/asdf-something/.bin:$PATH